geojson/__init__.py

```diff
@@ -0,0 +1,4 @@
+# SPDX-FileCopyrightText: 2024 Bryan Ramirez <bryan.ramirez@openenergytransition.org>
+#
```

Comment:
Usually, we put into the license header something quite general, like `PyPSA-Eur` authors to avoid a need to modify the header every time when the contributors list expands :D

From the other side, manuals of R packages provide name and contacts of the authors, and it works very well. So, we may probably keep it as is, and update along the way.